lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp

```diff
@@ -596,6 +608,32 @@ LLDBMemoryReader::getFileAddressAndModuleForTaggedAddress(
   return {{file_address, module}};
 }
 
+bool LLDBMemoryReader::readRemoteAddressImpl(
+    swift::remote::RemoteAddress address, swift::remote::RemoteAddress &out,
+    std::size_t size) {
+  assert((size == 4 || size == 8) &&
+         "Only 32 or 64 bit architectures are supported!");
+  auto *dest = (uint8_t *)std::malloc(size);
```

Comment:
If size is supposed to be 4 or 8, malloc is definitely overkill here. This could be stack allocated.